infra/modules/azure_core_infra/main.tf

```diff
@@ -136,5 +155,45 @@ module "dns" {
     name = module.network.vnet.name
   }
 
+  tags = var.tags
+}
+
+#------#
+# LOGS #
```

Comment:
why do we need this resource? (to log wehat?) I'd add a comment.

Reply:
is mandatory for github runner environment resource

Reply:
I'd add this as a comment

Reply:
I don’t think a comment is necessary. Yes, I added this resource because it’s essential for the runner, but it wasn’t added specifically for it, it’s a common resource. Adding a comment could create confusion, as it might be interpreted as “this is only for the GitHub runner,” which isn’t the case